packages/console/src/notebook/ScriptEditor.tsx

```diff
@@ -21,11 +21,15 @@ interface ScriptEditorProps {
   focusOnMount?: boolean;
   onChange: (e: editor.IModelContentChangedEvent) => void;
   onRunCommand: (command: string) => void;
+  onEditorInitialized: (editor: editor.IStandaloneCodeEditor) => void;
+  onEditorWillDestroy: () => void;
   session: IdeSession;
   sessionLanguage?: string;
   settings?: {
```

Comment:
Instead of defining our own type for settings, we should use the monaco editor type, `editor.IStandaloneEditorConstructionOptions`.